### ISSUE TITLE
Better handle the naming of multiple files for the same mod from the Nexus

### ISSUE THREAD

**Issue by izomiac** [ModOrganizer2/modorganizer#1720]
**Is your feature request related to a problem? Please describe.**
Mods on the Nexus often have a main mod file, then a some additional patches (e.g. the main plugin and an optional 4K texture pack for it).  I usually install them separately so I don't have to reinstall each patch in order when I update something.  However, when you install an updated file, the name that Mod Organizer 2 suggests is often not the same one you last installed it under.  For example, I might update "Dragon War - A Dragon Overhaul", but the suggested name is "Dragon War - Wyrmstooth Patch", or vice versa.  It is extremely easy to overlook the renaming step and accidentally install the file under the wrong name, which obviously causes problems.

**Describe the solution you'd like**
Mod Organizer 2 should track these optional files independently.  Suggest a name like "[Mod Name] - [Optional File Name]" and track version numbers by the file's title on the Nexus rather than by mod (since they are often different).

**Describe alternatives you've considered**
Another option is to check how many mods share the same Nexus ID, and if it's more than one then the naming step should have a special dialog box with the relevant mod names listed, so it's more obvious to the user that they need to pay special attention to the name.  This may be somewhat easier to implement, but is a less robust and more tedious solution overall.

![Screenshot](https://user-images.githubusercontent.com/6551261/178127179-7b1f1781-16a4-4222-a010-0e6b1779341d.png)

**Comment by suamor** [comment]
I fully agree. However MO2 version handling seems to be completely broken to me (at least for Skyrim):
1. The game version is ignored (do not force me to use latest version of skyrim all the time because it will crash my game)
2. It does not check game version with mod version
3. It shows updates for mods that are older than the current version (f.e. SSE Fixes)

I find myself to be forced to ignore updates on many files.
I will need to continue to manually downloading files and put in the download folder until this works.

(should be a critical problem, not a feature request IMO).
